src/traces/candlestick/attributes.js

```diff
@@ -0,0 +1,42 @@
+/**
+* Copyright 2012-2016, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+
+'use strict';
+
+var Lib = require('../../lib');
+var OHLCattrs = require('../ohlc/attributes');
+var boxAttrs = require('../box/attributes');
+
+var directionAttrs = {
+    name: OHLCattrs.increasing.name,
+    showlegend: OHLCattrs.increasing.showlegend,
+
+    color: Lib.extendFlat({}, boxAttrs.line.color),
+    width: Lib.extendFlat({}, boxAttrs.line.width),
```

Comment:
> Can we call these linecolor (in directionAttrs) and linewidth (in attributes)? 

sounds good.

>  it seems a bit funny to make a one-member container 

Agreed. I much prefer `increasing.linecolor` and `decreasing.lincolor`